web/yo/app/scripts/controllers/drugs.js

```diff
@@ -31,24 +61,40 @@ angular.module('oncokbApp')
 
             $scope.saveDrugName = function (newDrugName, drug) {
                 if (hasSameName(newDrugName, drug.uuid)) {
-                    modalError("Sorry", "Same name exists.", false, drug);
+                    modalError("Sorry", "Same name exists.", true, false, drug.uuid);
```

Comment:
modalError() has 6 parameters. In here, you only pass 5 so the last one is `null` by default?

Reply:
undefined. It works like realtimeString part.